components/questionnaire/Option.tsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+
+interface OptionProps {
+  answear: number;
+  className: string;
+  onClick(): void;
+}
+
+export function Option(props: OptionProps) {
```

Comment:
If you've got three props, maybe consider destructuring? 
`Option({className,onClick, answear}: OptionProps)`